tests/data/car.py

```diff
@@ -0,0 +1,113 @@
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the conditions stated in the LICENSE file in the project root for
+details.
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+"""
+
+import configsuite
+from configsuite import MetaKeys as MK
+from configsuite import types
+
+
+def build_schema():
+    return {
+        MK.Type: types.NamedDict,
+        MK.Content: {
+            "production_date": {MK.Type: types.String},
+            "tire": {
+                MK.Type: types.NamedDict,
+                MK.Required: False,
+                MK.Content: {
+                    "dimension": {
+                        MK.Type: types.Number,
+                        MK.Default: 17,
+                        MK.Required: False,
+                    },
+                    "rim": {
+                        MK.Type: types.String,
+                        MK.Default: "green",
+                        MK.Required: False,
+                    },
+                },
+            },
+            "color": {
+                MK.Type: types.NamedDict,
+                MK.Required: False,
+                MK.Content: {
+                    "chassis": {MK.Type: types.String},
+                    "interior": {MK.Type: types.String},
+                },
+            },
+            "owner": {
+                MK.Type: types.Dict,
+                MK.Required: False,
+                MK.Content: {
+                    MK.Key: {MK.Type: types.String},
+                    MK.Value: {
+                        MK.Type: types.NamedDict,
+                        MK.Content: {
+                            "name": {MK.Type: types.String},
+                            "location": {
+                                MK.Type: types.String,
+                                MK.Default: "Earth",
+                                MK.Required: False,
+                            },
+                        },
+                    },
+                },
+            },
+            "incidents": {
+                MK.Type: types.List,
+                MK.Required: False,
+                MK.Content: {
+                    MK.Item: {
+                        MK.Type: types.NamedDict,
+                        MK.Content: {
+                            "location": {MK.Type: types.String},
+                            "casualties": {
+                                MK.Type: types.Number,
+                                MK.Default: 0,
+                                MK.Required: False,
+                            },
+                        },
+                    }
+                },
+            },
+        },
+    }
+
+
+@configsuite.transformation_msg("Convert to cm")
+def _inch_to_cm(l):
```

Comment:
Please remove the prefix `_`, as you are planning to access it outside of this submodule.